src/mozmlops/templates/README.md

```diff
@@ -78,3 +78,81 @@ We know this workaround fails to account for scheduling and scripts: we're worki
 > [!NOTE]  
 > We have changed `offline` here to false: hat means we _do_ want our flow to integrate with Weights and Biases!
 
+## Running Inference Servers in Production
+
+Deploying your Ray Serve app to production requires changes in 3 different repositories: “application repo“, “function specific repo” and “deployment repo“ (please see [glossary](https://mozilla-hub.atlassian.net/wiki/spaces/DATA/pages/785514640/Deploy+inference+servers+to+production+GKE+using+Ray+Serve#Glossary) for details on these repositories).
```

Comment:
I think it's worth mentioning up front that, to get started and run an app _locally_, they only need to be in the application repo. It's daunting to think 3 repos will be needed up front; once the thing is working locally, _then_ they can make changes in the other two in order to permit deployment.

Reply:
I am adding a statement right after the current statement to clarify this.